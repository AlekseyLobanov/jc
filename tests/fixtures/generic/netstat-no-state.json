[{"proto":"udp","recv_q":0,"send_q":0,"local_address":"0.0.0.0","foreign_address":"0.0.0.0","state":null,"program_name":"avahi-daemon: r","kind":"network","pid":442,"local_port":"5353","foreign_port":"*","transport_protocol":"udp","network_protocol":"ipv4","local_port_num":5353},{"proto":"udp","recv_q":0,"send_q":0,"local_address":"0.0.0.0","foreign_address":"0.0.0.0","state":null,"program_name":"avahi-daemon: r","kind":"network","pid":442,"local_port":"49926","foreign_port":"*","transport_protocol":"udp","network_protocol":"ipv4","local_port_num":49926},{"proto":"udp6","recv_q":0,"send_q":0,"local_address":"::","foreign_address":"::","state":null,"program_name":"avahi-daemon: r","kind":"network","pid":442,"local_port":"5353","foreign_port":"*","transport_protocol":"udp","network_protocol":"ipv6","local_port_num":5353},{"proto":"udp6","recv_q":0,"send_q":0,"local_address":"::","foreign_address":"::","state":null,"program_name":"avahi-daemon: r","kind":"network","pid":442,"local_port":"35569","foreign_port":"*","transport_protocol":"udp","network_protocol":"ipv6","local_port_num":35569}]
