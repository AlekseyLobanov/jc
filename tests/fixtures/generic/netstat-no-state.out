Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
udp        0      0 0.0.0.0:5353            0.0.0.0:*                           442/avahi-daemon: r
udp        0      0 0.0.0.0:49926           0.0.0.0:*                           442/avahi-daemon: r
udp6       0      0 :::5353                 :::*                                442/avahi-daemon: r
udp6       0      0 :::35569                :::*                                442/avahi-daemon: r
